.github/workflows/pr-checks.yaml

```diff
@@ -0,0 +1,38 @@
+name: GitHub Action CI
+
+# We're using pull_request_target here instead of just pull_request so that the
+# action runs in the context of the base of the pull request, rather than in the
+# context of the merge commit. For more detail about the differences, see:
+# https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
+on:
+    pull_request_target:
+        # We don't need this to be run on all types of PR behavior
+        # See https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request
+        types:
+          - opened
+          - synchronize
+          - edited
+
+permissions: {} # none
+
+jobs:
+  check:
+    permissions:
+      pull-requests: write
+    name: Check Commits
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pull Request Commit Checker
+        uses: open-mpi/pr-git-commit-checker@v1.0.0
+        with:
+          token: "${{ secrets.GITHUB_TOKEN}}"
```

Comment:
Don't we need `cherry-pick-required: True` here?

Speaking of which, I guess I missed this on the `v4.1.x` and `v5.0.x` PRs.  Don't we need that on those branches, too?

Reply:
Dang, we do. Looks like I missed adding that when I re-cherry-picked the changes. It's present on the other branches however.

I'll add it here.